client/scss/components/_button.scss

```
@@ -151,6 +151,62 @@
 
     &.button-secondary {
       border-color: theme('colors.black-20');
+      color: $color-button;
+      background-color: transparent;
+      border: 1px solid $color-button;
+
+      .icon-wrapper {
+        border-right: 1px solid $color-button;
+        background-color: transparent;
+      }
+
+      &:hover {
+        background-color: theme('colors.secondary.50');
+        color: $color-button;
+      }
+
+      &:disabled,
+      &[disabled] {
+        background-color: transparent;
+        border-color: $color-grey-3;
+        color: $color-grey-3;
+        cursor: default;
+        pointer-events: none;
+
+        .icon-wrapper {
+          border-right: 1px solid $color-grey-3;
+          background-color: transparent;
+        }
+      }
+
+      &.button-small {
+        border: none;
+        padding: 0 30px 30px;
```

It looks like only the left padding is necessary here, and we should use `em` units when setting that, to be consistent with other padding values in this file.

```suggestion
        padding-inline-start: 2.25em;
```